crates/miden-lib/asm/miden/note.masm

```diff
@@ -336,3 +339,36 @@ export.add_note_assets_to_account
     # clear the stack
     drop dropw drop
 end
+
+#! Writes the assets data stored in the advice map to the memory by the provided destination
+#! pointer. 
```

Comment:
nit: I'd write this as: "Writes the assets data stored in the advice map to the memory **specified** by the provided destination pointer."